.github/workflows/wheel_noarch.yml

```diff
@@ -19,6 +19,9 @@ jobs:
         with:
           python-version: '3.7'
 
+      - name: Get latest CMake and ninja
+        uses: lukka/get-cmake@latest
```

Comment:
Is there any reason to prefer this over a pip installation of both cmake and ninja?